src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -299,26 +304,34 @@ def _update_upload_status_yaml_file(
     upload_env_key: str,
     workspace_id: int,
     load_id: str,
-    upa: str,
-    obj_tuple: _WSObjTuple,
+    assembly_upa: str,
+    assembly_tuple: _WSObjTuple,
```

Comment:
assembly_upa and assembly_tuple here are positional argument. So I assume if we upload genome, you will build assembly_tuple based on genbanks_to_genomes return?

Reply:
yea. I think we said genbanks_to_genomes will return assembly_ref as well. So we will always build assembly_tuple.